tests/integration/bytecode/test_kakarot.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+import web3
```

Comment:
these are WIP I guess

note: if you wanted to skip the test_cases, you could have just updated the `params_execute` list below